src/katsdpcontroller/generator.py

```diff
@@ -764,7 +764,7 @@ def get_baseline_index(a1, a2):
                default=stream.n_chans_per_substream, initial_status=Sensor.Status.NOMINAL),
         Sensor(float, f"{stream.name}.ddc-mix-freq",
                "F-engine DDC mixer frequency, where used. 0 where n/a",
-               default=0, initial_status=Sensor.Status.NOMINAL),
+               units="Hz", default=0.0, initial_status=Sensor.Status.NOMINAL),
```

Comment:
Ah, yes. I was actually looking around to see if the `sensor_type` arg was used, forgot to comment.

Reply:
That's the very first one.

Reply:
Argh, apologies. I always miss the first arg for some reason.